pkg/blueprint/blueprints/cockroachdb-blueprint.yaml

```
@@ -0,0 +1 @@
+/home/rajat/go/src/github.com/r4rajat/kanister/examples/cockroachdb/cockroachdb-blueprint.yaml
```

I think this should be the relative path
```suggestion
../../../examples/cockroachdb/cockroachdb-blueprint.yaml
```